bioblend/galaxy/tools/__init__.py

```diff
@@ -105,6 +105,14 @@ def requirements(self, tool_id):
         url = self._make_url(tool_id) + '/requirements'
         return self._get(url=url)
 
+    def reload(self, tool_id):
+        """
+        Reload specified tool.
```

Comment:
It's not updating the version necessarily.  It's updating the tool to be exactly what is in the wrapper.  Galaxy loads all of the tool wrappers at startup.  If a tool wrapper is changed, it will not be updated in the toolbox unless it is reloaded.

Reply:
Could you write one or two sentences to convey that in the docstring?